src/explanations/aave-other-features.yaml

```diff
@@ -47,4 +47,60 @@
    
     **Self Liquidation:** Markets can be very volatile and can lead to liquidation of the sensitive positions. Liquidation can be quite expensive on certain platforms. There are many applications that make use 
     of flash loans and liquidate the position on user’s behalf saving the user hefty liquidation fees of the protocol.
+ 
+
+
+- title: AAVE Portals
+  shortTitle: AAVE Portals
+  key: aave-portals
+  details: |
+   #### Layer 2 blockchains
+   In 2020, we had seen a lot of people using Ethereum, and transaction requests increased. However, Ethereum can only handle 30 transactions per second. This limitation caused gas prices to hike and transactions to become expensive.
+   On Ethereum mainnet, transactions can be slow and difficult, and the proof-of-work model can use a lot of energy. So we need an alternative that can improve these issues without compromising decentralization and security.
+   Layer 2 chains offer an alternative to the high transaction costs and slow speeds of Ethereum blockchain. The main reason Layer 2 blockchains exist is to increase the scalability of the blockchain. So the need for blockchain bridges arised.
+   
+
+   #### What are Blockchain Bridges and Why do we use them? 
+   Bridges in crypto act as a connection between two different blockchains. This connection is important because, without a bridge, blockchains would be isolated from each other.
+   However, with a bridge between two blockchains it becomes possible to transfer assets and arbitrary data between them. Thus, blockchain bridges promote the interoperability in the crypto ecosystem and are necessary to make different blockchain networks compatible.
+      
+      * **Disadvantages of bridges:**
+      Smart contract risks:
+      This can happen in both trusted and trustless bridge contracts. Bugs in the smart contract can lead it to be hacked and cause loss.
+      
+      * **Systematic financial risks:**
+      The majority of bridges use wrapped assets to mint canonical versions of original assets in the new chain. However, this exposes the bridge to systematic risk of the wrapped asset being exploited.
+      
+      * **Trust issues:**
+      Bridges use third-party for the transfer of assets, this will require the users to put trust on the validator to not engage in activities detrimental to the users, like rag pulling.
+      
+      * **Nascent stage:**
+      It is still unclear how bridges will work in various market conditions.
+   
```

Comment:
`Complexity` is another big issue as the user need to know the various steps needed to transfer the assets. And these steps are often not standard and varies from one bridge to another